server/src/test/java/org/elasticsearch/action/fieldcaps/FieldCapabilitiesResponseTests.java

```diff
@@ -152,6 +154,44 @@ public void testMixedVersionFailureSerialization() throws IOException {
         }
     }
 
+    /**
+     * check that failure serialization to minor versions before 7.13 works without serializing the failures part
+     */
+    public void testSerialization_pre_7_13() throws IOException {
+        FieldCapabilitiesResponse original = createResponseWithFailures();
+        FieldCapabilitiesResponse deserialized;
+        Version version = VersionUtils.randomVersionBetween(random(), Version.V_7_6_0, Version.V_7_12_1);
```

Comment:
I wonder if the upper bound should be before 7.13, or we may lose new bugfix in the 7.12 minor.

Reply:
The upper bound is inclusive here, so I'd like to pick the latest before 7.13. Maybe there's a helper for that.